packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -114,6 +114,8 @@ import { isString, isObject, PickOptions, QuickInputButtonHandle } from '@theia/
 import { Severity } from '@theia/core/lib/common/severity';
 import { DebugConfiguration, DebugSessionOptions } from '@theia/debug/lib/common/debug-configuration';
 
+import { CoverageDetails, IFileCoverage, ISerializedTestResults, RunTestForControllerRequest, TestsDiffOp, ITestRunProfile, ResolvedTestRunRequest, ITestItem, TestResultState, ITestMessage, ITestRunTask, ExtensionRunTestsRequest } from '@theia/testing/lib/common/test-types';
```

Comment:
It looks like you may need to add the `testing` package to the `package.json` for `plugin-ext` to make these items available in this package.

Reply:
The `tsconfig` should be updated automatically when running `yarn compile`.